src/test/parse_this_folder/specification/readme.md

```diff
@@ -0,0 +1,34 @@
+# Deployment Admin
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Deployment Admin.
+
+---
+
+## Getting Started
+
+To build the SDK for Deployment Admin, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+---
+
+```yaml $(tag) == 'package-2019-01-01'
+input-file:
+  - specs/a.json
+```
+
+---
+
+```yaml $(tag) == 'all-api-versions'
+require: $(this-folder)/../../../../profiles/readme.md
+
+# all the input files across all versions
+input-file:
+  - $(this-folder)/specs/b.json
```

Comment:
Prefer to call it `current-folder` for better understanding.

Reply:
Because in swagger readme.md file. it called 'this-folder'.